servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java

```diff
@@ -31,6 +31,15 @@
      */
     boolean hasOffloads();
 
+    /**
+     * Returns {@code true} if the instance has offloading for any operation.
```

Comment:
Looks like copy-paste. Please adjust the javadoc.

Reply:
+1, and add reference to `ListenableAsyncCloseable` methods to clarify what this flag contrils